### PR TITLE
Upgrade typescript and cut some weight from the packed extension

### PR DESCRIPTION
**Pull request by mtxr**
Describe here what is this PR about and what we are achieving merging this.

- Upgrade typescript and cut some weight from the packed extension

Like I mentioned before, there is code in the extension no longer being executed. Before releasing 0.25, I believe we should remove it.

----

Thank you for your contribution! 
Before submitting this PR, please make sure:

- [ ] Your code builds clean without any errors or warnings
- [ ] You have made the needed changes to the docs
- [ ] You have written a description of what is the purpose of this pull request above

### CODE REVIEW

File: packages/extension/package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "sqltools",
     "displayName": "SQLTools",
     "description": "Database management done right. Connection explorer, query runner, intellisense, bookmarks, query history. Feel like a database hero!",
-    "version": "0.25.0",
+    "version": "0.25.0-dev",
```
Comment (mtxr) [inline packages/extension/package.json]:
```suggestion
    "version": "0.25.0",
```

**Comment by gjsjohnmurray** [comment]
@mtxr I've pushed a few changes I needed to make so that the `pack:all` script would run on my Windows dev workstation. But I'm finding that the driver VSIXes are broken. For example if I install the 0.25.0 VSIX of the main extension and only the 0.4.0 VSIX of the SQLite driver (leaving the other bundled ones at the latest Marketplace versions, 0.3.0) then the 0.3.0 drivers register OK but the 0.4.0 one fails, Messages in the log:

```
[1663178193535] INFO  (ls): REQUEST RECEIVED => ls/RegisterPlugin
[1663178193535] INFO  (ls): request to register plugin: "c:\Users\JohnM\.vscode\extensions\mtxr.sqltools-driver-sqlite-0.4.0\out\ls\plugin.js"
[1663178193680] ERROR (ls): Error registering plugin: {}
    error: {}
[1663178193680] ERROR (ext): ERROR: Error loading plugin SQLite Plugin Request ls/RegisterPlugin failed with message: ls is not defined, {"code":-32603}
    ns: "error-handler"
[1663178194540] INFO  (ext): EXECUTING COMMAND => sqltools.refreshTree
[1663178194540] INFO  (ext): Connection explorer changed. Will be updated.
    ns: "conn-man:explorer"
[1663178204609] INFO  (ext): EXECUTING COMMAND => sqltools.showOutputChannel
```
The output from my `yarn run start` has this warning that looks significant:

```
ext:
  1 asset
  364 modules
  ext (webpack 5.74.0) compiled successfully in 11388 ms

ls:
  1 asset
  172 modules

  WARNING in ../packages/language-server/src/server.ts 64:53-60
  Critical dependency: require function is used in a way in which dependencies cannot be statically extracted

  ls (webpack 5.74.0) compiled with 1 warning in 11763 ms

ui:
...
```

Are you able to run the VSIXes you build?

**Comment by mtxr** [comment]
I know how to fix it. Let me take care

**Review by gjsjohnmurray — APPROVED** [review]
I bumped sqlite3 to 5.1.1 which claims not to need to build locally the binary for ARM Mac.

Driver VSIXes that I build locally on Windows still fail to load, so I built the two updated ones (MySQL 0.4.0 and SQLite 0.4.0) plus the main extension (0.25.0) on Linux (using GitHub Codespaces), tested them there, downloaded them and tested them locally on Windows. All looks good, so I'm approving this. I will then merge it, create the label and upload the three new VSIXes to Marketplace.

**Pull request merged**